### PR TITLE
blog: discovery of load balancer services via DNS

### PR DESCRIPTION
**Pull request by maiqueb**
**What this PR does / why we need it**:
This PR adds a blog post explaining how to allow users to discover load balancer service types (implemented by metalLB)  via DNS.

**Does this PR fix any issue?** _(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)_:

> Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: _posts/2023-03-06-expose-loadbalancer-servia-via-externaldns.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+author: Miguel Duarte Barroso
```
Comment (maiqueb) [inline _posts/2023-03-06-expose-loadbalancer-servia-via-externaldns.md:3]: how to add multiple authors ? :thinking:

File: _posts/2023-03-06-expose-loadbalancer-servia-via-externaldns.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+category: news
+tags:
+  [
+    "Kubevirt",
+    "kubernetes",
+    "virtual machine",
+    "VM",
+    "DNS",
+    "LoadBalancer"
+  ]
```
Comment (maiqueb) [inline _posts/2023-03-06-expose-loadbalancer-servia-via-externaldns.md:18]: consider adding metalLB here.

File: _posts/2023-03-06-expose-loadbalancer-servia-via-externaldns.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+TODO
+
+## Demo
+This post assumes the user has followed throught the installation instructions
+available in the
+[MetalLB KubeVirt blog post](https://kubevirt.io/2022/Virtual-Machines-with-MetalLB.html),
+and as a result already has `MetalLB` installed in the cluster. 
+
+### Configuring MetalLB
+TODO
```
Comment (maiqueb) [inline _posts/2023-03-06-expose-loadbalancer-servia-via-externaldns.md:40]: OK, both the kubevirt blog and our configuration use the layer2 mode. 

Do we want that ? Would we have to explain the caveats of l2 mode ? 

... I guess I'm asking if the demo would be clearer is we just used BGP configuration mode.

File: _posts/2023-05-06-expose-loadbalancer-servia-via-externaldns.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+## Demo
+This post assumes the user has followed throught the installation instructions
+available in the
+[MetalLB KubeVirt blog post](https://kubevirt.io/2022/Virtual-Machines-with-MetalLB.html),
+and as a result already has `MetalLB` installed in the cluster.
+
+### Architecture
+Please refer to the architecture diagram below to understand the solution.
+![MetalLB Service resolved by name](/assets/2023-05-08-metallb-external-dns/arch.jpg)
```
Comment (maiqueb) [inline _posts/2023-05-06-expose-loadbalancer-servia-via-externaldns.md:41]: uff this renders awful (too big).

Must find a way to limit this in size.

**Comment by cwilkers** [comment]
Hi @maiqueb , do you need some help with this? It still has do-not-merge/work-in-progress, but has now aged out to rotten and will be closed shortly by the bot.

**Comment by maiqueb** [comment]
Pretty much, but I'm facing now an issue: coming up w/ the dynamic DNS updates tooling is pretty far from my current knowledge set .

I'd need to sort that out before I can continue. 

Maybe letting the bot close this is OK; I'll re-open once I have enough buffer room to resume working on it.

Thanks for asking @cwilkers

**Pull request closed without merging**